tests/plugins/test_subcommands.py

```diff
@@ -1,23 +1,30 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
+from unittest import mock
+
```

Comment:
No need to import mock here since `mocker.call` also works, https://pytest-mock.readthedocs.io/en/latest/usage.html

Reply:
Neat! Thanks for the tip.